geode-core/src/distributedTest/java/org/apache/geode/management/JMXMBeanReconnectDUnitTest.java

```diff
@@ -86,6 +86,11 @@
   @Before
   public void before() throws Exception {
     locator1 = lsRule.startLocatorVM(LOCATOR_1_VM_INDEX, locator1Properties());
+
+    System.out.println("GEODE-6107: waiting for locator to start");
```

Comment:
Are `System.out.println` calls here to further diagnose this bug, or were they just forgotten to be removed?  If they're meant to be kept, I'd appreciate a comment mentioning it.

But, *this time* the beans will behave, I bet.

Reply:
oops. I'll remove that. Thank you.

Reply:
And yes, now the beans behave. I ran this before block over 4000 times without failure.